examples/purchase-tester/src/main/AndroidManifest.xml

```diff
@@ -20,4 +20,7 @@
             </intent-filter>
         </activity>
     </application>
+    <queries>
+        <package android:name="com.amazon.sdktestclient" />
+    </queries>
```

Comment:
I was thinking about moving this to the `feature/amazon` package manifest, but that's probably a bad idea since we would make our app's users potentially have this as well. So this is good 👍